applications/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,84 @@
+.. _suit_flash_companion:
+
+SUIT flash companion
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The flash companion application allows the secure domain firmware to access the external memory during the SUIT firmware upgrade.
```

Comment:
Add link to SUIT firmware update documentation in the NCS.

Reply:
Added link to the documentation about SUIT.